pkg/cmd/server/origin/master_config.go

```diff
@@ -575,19 +575,19 @@ func newAdmissionChain(pluginNames []string, admissionConfigFilename string, plu
 			plugin = saAdmitter
 
 		default:
-			configFile, err := pluginconfig.GetPluginConfigFile(pluginConfig, pluginName, admissionConfigFilename)
+			configFile, err := pluginconfig.GetAdmissionConfigurationFile(pluginConfig, pluginName, admissionConfigFilename)
```

Comment:
I expected this to get a lot simpler with the upstream changes from @derekwaynecarr... no weird temp files needed, etc